src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage?: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePrev = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNext = () => {
+    if (currentPage < totalPages) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
```

Comment:
You're using a `href` here, which will cause the page to jump to the top when clicked. This can be a bit jarring for the user. To prevent this, you could use `href="javascript:void(0);"` instead. This will prevent the default action of the click.